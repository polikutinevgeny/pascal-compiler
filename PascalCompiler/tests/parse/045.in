var
   a, i: integer;
begin
   for i := 10 to n + 5 do0
      a := i*j;
end.