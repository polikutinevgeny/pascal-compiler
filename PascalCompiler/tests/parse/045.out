Expected Do, got Identifier at 4:25
